web-next/app/google/antigravity-guide/page.module.css

```
@@ -1,1509 +1,550 @@
 
-.artName {
-  font-family: var(--font-display, "Syne", sans-serif);
-  font-size: 0.85rem;
-  font-weight: 600;
-  color: #ddeeff;
+/*
+ * diag-3（実行モード・3状態循環）と diag-5（サブエージェントライフサイクル・5状態）は
+ * stateDiagram-v2 でノード数が少なく viewBox が広くなるため文字が巨大化する。
+ * 他の図解（graph TD / flowchart LR / sequenceDiagram）は自然サイズで問題なし。
+ */
+:global(#diag-3 svg) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)